src/main/java/Deadline.java

```diff
@@ -0,0 +1,17 @@
+public class Deadline extends Task {
+    private final String by;
```

Comment:
```suggestion
    private final String BY
```

perhaps you could try changing the naming of the variable to the above? I notice similar issues in other places